extensions/amp-access/0.1/jwt.js

```diff
@@ -103,3 +177,79 @@ function decodeBase64WebSafe(s) {
 function unmapWebSafeChar(c) {
   return WEB_SAFE_CHAR_MAP[c];
 }
+
+
+/**
+ * Convers a text in PEM format into a binary array buffer.
+ * @param {string} pem
+ * @return {!ArrayBuffer}
+ * @visibleForTesting
+ */
+export function pemToBinary(pem) {
+  // TODO(dvoytenko, #4281): Extract with other binary encoding utils (base 64)
+  // into a separate module.
+  pem = pem.trim();
+  let start = 0;
+  let end = pem.length;
+
+  // Remove pem prefix, e.g. "----BEING PUBLIC KEY----".
+  if (pem.charAt(0) == '-') {
```

Comment:
We can to some extent. "BEGIN" and "END" are required here. The exact words inside are a bit open-ended, but can be munched through with regexp. I'll try.